example/src/aco.rs

```diff
@@ -1,21 +1,15 @@
 use ecrs::aco::{probe::CsvProbe, AntSystemCfg};
 
-
 pub fn ants_example_run() {
-  let (cities, cost) = ecrs::aco::generate_tsp_cost(30);
-  ecrs::aco::write_cities_csv(&cities, "cities.csv").expect("Error while writing city file");
+    let (cities, cost) = ecrs::aco::generate_tsp_cost(30);
+    ecrs::aco::write_cities_csv(&cities, "cities.csv").expect("Error while writing city file");
```

Comment:
We must enforce one indentation level across whole project. 
It should be done in #92 - as I believe that `fmt` tool can achieve this. 
I'll conduct a small survey on our discord.